### ISSUE TITLE
operation.requestBody.content must not be null

### ISSUE THREAD

**Issue by dextersjab** [specmatic/specmatic#529]
**Description**
Load Error FAILED
Reason: operation.requestBody.content must not be null

**Steps to reproduce**
Ran `specmatic test ./openapi.yaml` against my API spec, containing the below:

```yaml
...
requestBody:
  $ref:
    '#/components/requestBodies/RegistrationUpdates'
...
```

**Expected behavior**
Expected the test to run.

Instead, I received the below error:

```shell
operation.requestBody.content must not be null
Load Error FAILED
Reason: operation.requestBody.content must not be null



Unsuccessful Scenarios:
  "Load Error FAILED"
        Reason: operation.requestBody.content must not be null

Tests run: 1, Successes: 0, Failures: 1, Errors: 0

```

**System details**
java --version                
openjdk 16.0.1 2021-04-20
OpenJDK Runtime Environment AdoptOpenJDK-16.0.1+9 (build 16.0.1+9)
OpenJDK 64-Bit Server VM AdoptOpenJDK-16.0.1+9 (build 16.0.1+9, mixed mode, sharing)

Specmatic version: 0.51.0

**Additional context**

Might be related to #528.

**Comment by harikrishnan83** [comment]
@dextersjab Thanks for reporting the issue. Also the details you shared helped us reproduce the problem and write a test for the same. We have fixed this issue and this should be part of our next release.

**Comment by harikrishnan83** [comment]
@dextersjab we have released [0.53.0](https://github.com/znsio/specmatic/releases/tag/0.53.0) with a fix for this issue. Closing this issue. Please re-open if necessary. Thanks.
